fix/expr_data/rrfs-data_fv3jedi_2022052619

```diff
@@ -0,0 +1 @@
+../.agent/../rrfs-test-data/rrfs-data_fv3jedi_2022052619
```

Comment:
@guoqing-noaa  I am a little lost on the structure now :).  Can we stay away from using .agent for being now or leave this kind of changes to separate PR for a more focused discussion?

Reply:
@guoqing-noaa I agree with @TingLei-NOAA , we need another issue and PR to address this so that the developers can know this.

Reply:
@ShunLiu-NOAA @TingLei-NOAA 

We have an open issue for this: issue #93 .  This PR is to address that problem partly.  But it is not that descriptive and this PR title is not consistent. Let me make some changes and get them consistent and descriptive.

Reply:
@ShunLiu-NOAA @TingLei-NOAA @delippi @SamuelDegelia-NOAA 
FYI, issue #93 is updated to be more clear and the title and description of this PR are also updated. Thanks!

Reply:
I created issue #157 to address the '.agent' concerns